src/components/Calendar.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div id="calendarFull">
+    <button @click="refreshEvents">Refresh</button>
+    <button v-if="selected._id" @click="removeEvent">Remove</button>
+    <pre v-if="selected._id">Title: {{selected.title}}
+    Start: {{selected.start}}
+    End: {{selected.end}}
+    Allday: {{selected.allDay}}
+    ID: {{selected.id}}</pre>
+    <full-calendar ref="calendar" :event-sources="eventSources" @event-drop="eventDrop" @event-resize="eventResize" @event-selected="eventSelected" @event-created="eventCreated" :config="config"></full-calendar>
+  </div>
+</template>
+
+<script>
+import moment from 'moment';
+import frappe from "frappejs";
+
+const getCircularReplacer = () => {
+  const seen = new WeakSet;
+  return (key, value) => {
+    if (typeof value === "object" && value !== null) {
+      if (seen.has(value)) {
+        return;
+      }
+      seen.add(value);
+    }
+    return value;
+  };
+};
+
+export default {
+  name: 'calendarFull',
+  data() {
+    return {
+      events: [
```

Comment:
query this via `frappe.db.getAll`

Reply:
I've done that below, need to clean the code a bit, will get that done today.